### PR TITLE
docs(ecosystem): Add fastify-list-routes

### PR DESCRIPTION
**Pull request by chuongtrh**
Added **fastify-list-routes** to the ecosystem core plugins.

I thought this interesting module is worth adding to the ecosystem plugins page, so that the community is more aware of it.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -608,6 +608,7 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`fastify-list-routes`](https://github.com/chuongtrh/fastify-list-routes) A simple plugin for Fastify list all available routes.
```
Comment (climba03003) [inline docs/Guides/Ecosystem.md:611]: Can you move into `Community Plugin`? Core plugin means the plugins maintained inside the `Organization`.

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: docs/Guides/Ecosystem.md (hunk 1 above)
Comment (climba03003) [inline docs/Guides/Ecosystem.md:611]: Can you follow the alphabetical order?

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: docs/Guides/Ecosystem.md (hunk 1 above)
Comment (chuongtrh) [reply docs/Guides/Ecosystem.md:611]: Yes, will do

File: docs/Guides/Ecosystem.md [hunk 2]
```diff
@@ -378,7 +378,8 @@ section.
   Fastify plugin to parse request language.
 - [`fastify-lcache`](https://github.com/denbon05/fastify-lcache)
   Lightweight cache plugin
-- [`fastify-list-routes`](https://github.com/chuongtrh/fastify-list-routes) A simple plugin for Fastify list all available routes.
+- [`fastify-list-routes`](https://github.com/chuongtrh/fastify-list-routes) 
+A simple plugin for Fastify list all available routes.
```
Comment (climba03003) [inline docs/Guides/Ecosystem.md:382]:
```suggestion
  A simple plugin for Fastify list all available routes.
```

**Review by climba03003 — APPROVED** [review]

**Review by Eomm — APPROVED** [review]

**Pull request merged**